frontend/src/types/SelectInputTypes.ts

```diff
@@ -0,0 +1,10 @@
+import { OptionBase } from "chakra-react-select";
```

Comment:
can we name these smth else? e.g. StringOption and StringOrBoolOption

Reply:
Will do!